mitmproxy/utils/typecheck.py

```diff
@@ -26,7 +26,7 @@ def check_type(attr_name: str, value: typing.Any, typeinfo: type) -> None:
     if typename.startswith("typing.Union"):
         try:
             types = typeinfo.__args__
-        except AttributeError:
+        except AttributeError:  # pragma: no cover
```

Comment:
Presuming you mean attribute access and not import: yes - so ideally we'd have a mock object that triggers this specific case instead of taking it out of coverage?